nntrainer/src/tensor.cpp

```diff
@@ -238,6 +239,15 @@ int Tensor::subtract_i(float const &value) { return this->add_i(-value); }
 
 Tensor Tensor::subtract(float const &value) { return this->add(-value); }
 
+Tensor Tensor::pow(float exponent) const {
+  Tensor result(dim);
+
+  for (unsigned int i = 0; i < length(); ++i)
+    *result.getAddress(i) = powf(*getAddress(i), exponent);
```

Comment:
use std::transform than a loop here.
Also, i wonder about the accuracy of pow for the exponent of 2 vs using cblas multiply? Is using `powf` more accurate?

Reply:
Accuracy is verified with (0.5f and `sqrtFloat`)
This is for negative and non-integer power.

Updated to use apply instead